cloudinit/config/cc_ntp.py

```diff
@@ -364,21 +379,29 @@ def generate_server_names(distro):
     """
     names = []
     pool_distro = distro
-    # For legal reasons x.pool.sles.ntp.org does not exist,
-    # use the opensuse pool
     if distro == 'sles':
+        # For legal reasons x.pool.sles.ntp.org does not exist,
+        # use the opensuse pool
         pool_distro = 'opensuse'
+    elif distro == 'alpine':
+        # Alpine-specific pool (i.e. x.alpine.pool.ntp.org) does not exist
+        # so use general x.pool.ntp.org instead.
+        pool_distro = ''
+
     for x in range(0, NR_POOL_SERVERS):
-        name = "%d.%s.pool.ntp.org" % (x, pool_distro)
-        names.append(name)
+        names.append(".".join([host for host in
+                          [str(x) + [pool_distro] + ['pool.ntp.org'] if host]))
```

Comment:
```suggestion
                          [str(x), pool_distro, 'pool.ntp.org'] if host]))
```

Reply:
I had a missing "]" after the str(x). Fixed now.